lms/djangoapps/mobile_api/users/views.py

```diff
@@ -323,7 +329,8 @@ def get_queryset(self):
             enrollment for enrollment in enrollments
             if enrollment.course_overview and self.is_org(org, enrollment.course_overview.org) and
             is_mobile_available_for_user(self.request.user, enrollment.course_overview) and
-            not self.hide_course_for_enrollment_fee_experiment(self.request.user, enrollment)
+            not self.hide_course_for_enrollment_fee_experiment(self.request.user, enrollment) and
+            check_course_expired(self.request.user, enrollment.course) == ACCESS_GRANTED
```

Comment:
I might add a test that a course is added when it has not expired and is not added when it has expired.

Reply:
good idea